src/prices/prices.controller.ts

```diff
@@ -19,9 +20,26 @@ export class PriceController {
   @Description('Return a map of checksum contract address to the currency value of the token')
   @Returns(200, PriceSummaryModel)
   async listPrices(
+    @QueryParams('tokens') tokens?: string[],
```

Comment:
The token value doesn't seem to be used, the query will always be for all the chain tokens. Maybe use a flag?

Reply:
its used on line 27